hadoop-ozone/s3gateway/pom.xml

```diff
@@ -147,48 +147,6 @@
       <groupId>org.apache.ozone</groupId>
       <artifactId>hdds-hadoop-dependency-test</artifactId>
       <scope>test</scope>
-      <exclusions>
```

Comment:
is this a related change?

Reply:
Yes, it's related, but not strictly required.  I was trying to eliminate `log4j` coming as transitive dependency via various Hadoop test jars e.g. `hadoop-hdfs`.  I replaced dependence on `hadoop-hdfs` with `hdds-hadoop-dependency-test`.  Since the latter already excludes transitive dependencies, these local exclusions seemed unnecessary.